routes/index.js

```diff
@@ -1,67 +1,144 @@
 /*eslint no-undef: "error"*/
 /*eslint no-console: ["error", { allow: ["warn", "error"] }] */
 /*eslint-env node*/
-
+// ------ set up ------
 const express = require('express');
 const router = express.Router();
-
 const alasql = require('alasql');
 require('../item_db.js');
+require('../config/auth.js');
+const passport = require('passport');
+const bodyParser = require('body-parser');
+const session = require('express-session');
+const bunyan = require('bunyan');
+// ----- configuration -----
+router.use(require('morgan')('combined'));
+router.use(bodyParser.json());
+router.use(session({ secret: 'keyboard cat', resave: false, saveUninitialized: true }));
+router.use(passport.initialize());
+router.use(passport.session());
+const log = bunyan.createLogger({name: 'router'});
+
+// ----- funcion for checking  -----
+function isLoggedIn(req, res, next) {
```

Comment:
your `index.js` is getting too long.
please break up your code to modules, based on URI namespace:
`/item/*` -> `./routes/item.js`
`/profile/*` -> `./routes/profile.js`
`/log*/*` and middleware -> `./auth.js`

Reply:
routes are now living in three files items, profile and authentication. Middleware was move into middlewares.js file in config folder.